pages/settings/interface/index.vue

```diff
@@ -14,29 +14,29 @@ useHydratedHead({
       </div>
     </template>
     <div p6 flex="~ col gap6">
-      <div space-y-2>
-        <p font-medium>
+      <label space-y-2>
```

Comment:
this should be a div, html markup is wrong, cannot add p, div, or button inside a label

Reply:
Sorry, maybe I made a mistake during merge

Reply:
Let me revert to `<div>`

Reply:
iirc this should be fixed in another PR, maybe merge problem?

Reply:
OK, fixed it now.